src/reference/modules/usage/pages/variables.adoc

```diff
@@ -84,26 +85,30 @@ For properties with a common name, an override is done with the following algori
 - if both values are JSON array, child array is kept,
 - if both values are JSON object, object are merged using current algorithm recursively: missing keys are added, etc. 
 
-[TIP]
+You can choose the behavior for each of string, array or object cases independently and at each level (global parameters by
+UI or API, group properties and node properties by API) with a string of tree letters alike file permissions string: 
```

Comment:
Is something planned for a specific interface? This this form is good for storage and API but Web interface could be more visual.

Reply:
Yes of course we will need an UI for that. It was too short for 6.2, so we want to the main use case (ie the one explained by users): change perm for one property (on the whole hierarchy).
I will need to be done in 7.0 (and also: we can make something better in UI than let the user enter 3 letters, like dropdowns or two/three-state buttons)